docs/bot-usage.md

```diff
@@ -106,7 +106,7 @@ user_data/
 ├── backtest_results
 ├── data
 ├── hyperopts
```

Comment:
This seems kinda misaligned ...

maybe we should rename this folder too? (and the corresponding load-scripts ...) ?

Reply:
do not think so: `strategies` --> `hyperopts`

Reply:
`grep -ri hyperopt freqtrade tests docs'  -- there are many 'hyperopts' left (where it's convenient and correct)